GA/problem_definition.py

```
@@ -0,0 +1,465 @@
+                      "tflow": "tflow.{}",
+                      "forces": "forces.{}",
+                      "flowfield": "flowfield.{}",
+                      "boundary_layer": "boundary_layer.{}",
+                      "tdat": "tdat.{}"}
+
+    # Initialize output dictionary to use in case of an infeasible design.
+    # This equals the outputs of the output_handling.output_processing.GetAllVariables(3) method,
+    # but is quicker as it does not involve reading a file.
+    CRASH_OUTPUTS: dict[str, dict[str, float] | dict[str, dict[str, float]]] = {'data':
+                                                                                {'Total power CP': 0.00000,
+                                                                                'EtaP': 0.00000,
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)